airflow/providers_manager.py

```diff
@@ -905,7 +905,7 @@ def _discover_filesystems(self) -> None:
 
     def _discover_asset_uri_resources(self) -> None:
         """Discovers and registers asset URI handlers, factories, and converters for all providers."""
-        from airflow.assets import normalize_noop
+        from airflow.sdk.definitions.asset import normalize_noop
```

Comment:
It’s somewhat weird this imports from the SDK. I don’t have very good ideas though.

Reply:
looks like we're not doing anything special https://github.com/apache/airflow/blob/c3aabba04881e5d25fe3b6ef7727fb52d28b8a2c/airflow/providers_manager.py#L939. so maybe we can just make it a lambda?

Reply:
It’s not really better though. Let’s keep it this way.